Makefile

```diff
@@ -147,21 +147,6 @@ deps:
 
 # artifacts
 
```

Comment:
Opening a thread in case there's discussion RE: 

> It should also be possible to register a single handler for all endpoints that begin with /v2 instead of keeping all the existing routes.

* I agree with @algochoi's remark in https://github.com/algorand/go-algorand/pull/4847#issuecomment-1339999622.
* I think the as is PR reflects the desired behavior.

Reply:
Resolving - From an out-of-band verbal, we've agreed to keep as is.